src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -678,12 +682,63 @@ void GraphOptimizer::FuseMultiplyAndAdd(Graph& graph) {
     }
 }
 
-void GraphOptimizer::MergeConvertAndScaleShift(Graph& graph) {
+void GraphOptimizer::MergeEltwiseAndConvert(Graph& graph) {
+// The pass is required on arm platforms only
```

Comment:
Not really, it might be usefull for other platforms as well (but it requires additional perf validation). Lets create corresponding ticket and put the numebr here.

Reply:
Changed the comment and added ticket number